agents/xy_stage/xy_latrt_agent.py

```diff
@@ -191,7 +191,7 @@ def start_acq(self, session, params=None):
                 data['data']['y'] = pos[1] 
 
                 self.agent.publish_to_feed('positions',data)
-
+                session.data.update( data['data'] )
```

Comment:
Accompanying this should be a docstring showing the structure of data in `session.data`. The contents of `session.data` aren't necessarily exactly the 'data' dict passed to the OCS feed, so when someone's writing a client they can refer to this docstring in the Sphinx docs to know what's available.

An example can be seen in the ocs docs: https://ocs.readthedocs.io/en/develop/developer/session-data.html#agent-access